sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -349,5 +366,30 @@ public IReadOnlyList<FormField> AsList()
 
             return fieldDictionary;
         }
+
+        /// <summary>
+        /// Gets the value of the field as a <see cref="FormSelectionMarkState"/>.
+        /// </summary>
+        /// <returns>The value of the field converted to <see cref="FormSelectionMarkState"/>.</returns>
+        /// <exception cref="InvalidOperationException">Thrown when <see cref="ValueType"/> is not <see cref="FieldValueType.SelectionMark"/>.</exception>
+        public FormSelectionMarkState AsFormSelectionMarkState()
+        {
+            if (ValueType != FieldValueType.SelectionMark)
+            {
+                throw new InvalidOperationException($"Cannot get field as SelectionMark.  Field value's type is {ValueType}.");
+            }
+
+            if (_fieldValue == null)
+            {
+                return ValueSelectionMark;
+            }
+
+            if (!_fieldValue.ValueSelectionMark.HasValue)
+            {
+                throw new InvalidOperationException($"Field value is null.");
+            }
+
+            return _fieldValue.ValueSelectionMark.Value;
```

Comment:
I might be commenting in the wrong place so correct me if I missed it. I remember that the service team said that it was a bug that valueSelectionMark.value was returning null and that it wouldn't be fixed until 2.1-preview.3. I'm thinking that in the SDK we should return the value found in `text` here in the meantime. What do you think?

Reply:
hmmm we could. I was treating it as we did with the PhoneNumber fieldValue where we return what the service returns.
I am ok making the change and putting a TODO to delete once we confirm the service fixed it

Reply:
Code updated to reflect that